code/API_definitions/device-reachability-status-subscriptions.yaml

```diff
@@ -401,19 +401,19 @@ components:
     ErrorInfo:
       type: object
       required:
+        - message
```

Comment:
I understood from https://github.com/camaraproject/Commonalities/issues/515 that we should keep the inital order.